scripts/format_notebooks.py

```diff
@@ -520,7 +520,7 @@ def preprocess(self, nb, resources):
                     check_failed.append(str(file_loc))
 
                     if on_ci:
-                        # CI doesn't provide enough state to diagnose a peculiar or
+                        # Buildkite CI doesn't provide enough state to diagnose a peculiar or
```

Comment:
```suggestion
                        # CI doesn't provide enough state to diagnose a peculiar or
```

Github Actions doesn't provide enough state either.